lib/reporter.js

```diff
@@ -202,18 +241,36 @@ class Reporter {
             .promise.catch(errorHandler);
     }
 
+    // eslint-disable-next-line class-methods-use-this
+    getTime () {
+        return new Date().valueOf();
```

Comment:
Why not to use `this.client.helpers.now`?

Reply:
The client's method is not quite suitable, since `getTime` method works faster than the client's one